src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -0,0 +1,51 @@
+import { Link, Skeleton, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchRegistration } from '../../../api/registrationApi';
+import { Registration, RegistrationStatus } from '../../../types/registration.types';
+import { StyledStatusMessageBox } from '../../messages/components/PublishingRequestMessagesColumn';
+
+interface DeletedRegistrationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+  const { t } = useTranslation();
+  const duplicateRegistrationQuery = useQuery({
+    enabled: !!registration.duplicateOf,
+    queryKey: ['registration', registration.duplicateOf],
+    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const duplicateRegistrationTitle = duplicateRegistrationQuery.data?.entityDescription?.mainTitle;
+  const unpublishingNote = registration.publicationNotes?.find((note) => note.type === 'UnpublishingNote');
+
+  return (
+    <StyledStatusMessageBox
+      sx={{
+        bgcolor: 'publishingRequest.main',
+      }}>
+      {(registration.status === RegistrationStatus.Unpublished ||
+        registration.status === RegistrationStatus.Deleted) && (
+        <Typography variant="h4" component="span" sx={{ textTransform: 'uppercase' }}>
+          {t(`registration.status.${registration.status}`)}
+        </Typography>
+      )}
+
+      {unpublishingNote?.note && <Typography>{unpublishingNote.note}</Typography>}
+      {registration.duplicateOf && (
+        <Typography aria-busy={duplicateRegistrationQuery.isFetching} aria-live="polite">
```

Comment:
Ville fjerna spesiell styling her, sånn at alle boksene blir like. Skissene for dette er ulike her-og-der, men det burde uansett være `StyledStatusMessageBox` som sikrer at alle meldinger presenteres noenlunde likt, og at det ikke varierer mellom lowercase/uppercase etc ulike plasser

```suggestion
        <Typography>{t(`registration.status.${registration.status}`)}</Typography>
      )}

      {unpublishingNote?.note && <Typography sx={{ gridColumn: '1/3' }}>{unpublishingNote.note}</Typography>}
      {registration.duplicateOf && (
        <Typography sx={{ gridColumn: '1/3' }} aria-busy={duplicateRegistrationQuery.isFetching} aria-live="polite">
```

Reply:
rotete diff igjen da.. men har:
- fjernet custom styling på status, slik at den blir lik alt annet
- lagt til at `note` og `duplicateOf` skal ta bredden til hele grid row, for å unngå litt kunstig alignment og linjeskift